bdsf/opts.py

```diff
@@ -73,6 +73,7 @@ class Opts(object):
     filename = String(doc = "Input image file name\n"\
                                  "The input image can be a FITS or CASA 2-, "\
                                  "3-, or 4-D cube.")
+    logfilename = String(doc = "Log filename")
```

Comment:
I think it would make more sense to put this option in the "outputs options" section (starting around line 858 in `opts.py`). We should also match the way the other optional string options are defined -- so something like:
```
    logfilename = Option(None, String(),
                             doc = "Filename of the log file. None => get "\
                                 "from filename",
                             group = "output_opts")
```
However, this also raises the issue that if one activates many of the output options (such as `savefits_rmsim` or `output_all`), PyBDSF will still try to write to the input image directory. So perhaps we should make a more general solution, allowing the user to specify the output directory where the log file and other outputs would go? This could mirror the `indir` option that's already there.

Reply:
Ah great idea. I'll have another look.

Reply:
Ah, wait. What would the output directory be, with the current implementation on master, if we specify `indir`? Would it maybe go to the current working directory? That would give a good workaround, without modifying any code

Reply:
Ah, good thought -- unfortunately, as far as I can tell, it is not used that way (see `readimage.py`, around line 45): `indir` seems only to be used for the reading of the image data and header (I can't remember who added this option or why, probably me :) ).